app/views/snippets/equipment-list.liquid

```diff
@@ -30,9 +30,11 @@
               <a href="{% path_to child %}" title="Pick up your {{ child.title }}">
                 <button class="">Pick up Now</button>
               </a>
+              {% if libcal_url %}
```

Comment:
Nice...you could use [liquid's `capture`](http://shopify.github.io/liquid/tags/variable/#capture) here to simplify this a bit.

Review [this example from the `form-site-feedback` snippet](https://github.com/cul-it/mann-wagon/blob/474378592edd88b5a790717ab1294165220991e9/app/views/snippets/form-site-feedback.liquid#L4-L10) that includes conditionals within the capture:

```liquid
{% capture page_in_question %}
  {% if post? %}
    {{ site_feedback.url }}
  {% else %}
    {{ referer }}
  {% endif %}
{% endcapture %}
```